app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Welcoming Ukrainian teachers and trainee teachers to the UK"
+description: |-
+  Advice for Ukrainian refugees who wish to teach or train to teach in England.
+related_content:
+    Get support training to teach if you're disabled: "/funding-and-support/if-youre-disabled"
+    Teach in England if you trained overseas : "/non-uk-teachers/teach-in-england-if-you-trained-overseas"
+    Who do you want to teach? : "/train-to-be-a-teacher/who-do-you-want-to-teach"
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+navigation: 20.15
+navigation_title: Welcoming Ukrainian teachers and trainee teachers to the UK
+navigation_description: Advice for Ukrainian citizens seeking teaching jobs and training places in England
```

Comment:
Expand on this a little. Change to:
Find information and advice for Ukrainian citizens seeking teaching jobs and training places in England